README.md

```diff
@@ -19,9 +19,7 @@ pycall.rb supports Ruby version 2.4 or higher.
 
 ## Supported Python versions
 
-pycall.rb supports Python version 2.7 or higher.
-
-Note that in Python 2.7 old-style class, that is defined without a super class, is not fully supported in pycall.rb.
+pycall.rb supports Python version 3.7 or higher.
```

Comment:
@mrkn should this say 3.6 or higher? it looks like you are still testing python 3.6: https://github.com/mrkn/pycall.rb/blob/f5c2c2f762094c0b42e661dc8662b97779db9949/.github/workflows/ci.yml#L50